jhu/delphi_jhu/run.py

```diff
@@ -137,3 +147,15 @@ def run_module():
         # Report failures: someone should probably look at them
         for exported_file in fails:
             print(f"Failed to archive '{exported_file}'")
```

Comment:
Is mixing prints and logging here intentional?

Reply:
It's intentional in that I didn't actively replace print statements yet... fine for now.